terraform/security_group.tf

```diff
@@ -10,7 +15,21 @@ resource "aws_security_group" "db-security-group" {
   }
 }
 
-resource "aws_security_group_rule" "remote-db-access" {
+output "aws_security_group_id" {
+  value = aws_security_group.eb-security-group.id
+}
+
+resource "aws_security_group_rule" "eb-db-access" {
+  security_group_id        = aws_security_group.db-security-group.id
+  type                     = "ingress"
+  description              = "Access given to the security group applied to all EB instances"
+  from_port                = 5432
+  to_port                  = 5432
+  protocol                 = "tcp"
+  source_security_group_id = aws_security_group.eb-security-group.id
+}
+
+resource "aws_security_group_rule" "remote-db-access-ttung" {
```

Comment:
`ttung`?

Reply:
yup, this is a rule that allows my ip address at home to access the instance.  feel free to add one for your ip.

Reply:
TBH I'm not a fan of this, I think it would be better to set up a bastion host and use an ssh tunnel.

Reply:
I'll remove this from this PR.  Honestly, I think it's not a big deal, but I'd rather move forward.